lib/service/endpoint.ml

```diff
@@ -151,7 +151,8 @@ let form_action_with = href_with
 let form_action = href
 
 let form_method
-  : (_, Method.for_form_action, _, _) wrapped -> [> Method.for_form_action ]
+  : type scope.
+    (scope, Method.for_form_action, _, _) wrapped -> [> Method.for_form_action ]
```

Comment:
Just for my understanding and personal knownledge, why is this mandatory ?

Reply:
Oh, there is a quantifier! Nevermind!

Reply:
Yep